change/@ni-nimble-blazor-ebde87c2-e71d-40c0-bff5-f948c6b305dd.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Change data API of Nimble Blazor Table.",
```

Probably worth adding a note here to point people to the nimble-blazor README.md, since you put the details on the updated API there.

---

And could be more specific too.
```suggestion
  "comment": "BREAKING CHANGE: Nimble Blazor Data bindable property changed to SetDataAsync method . See Nimble Blazor README for usage info.",
```